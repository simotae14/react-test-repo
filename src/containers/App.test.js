import { shallow } from 'enzyme';
import React from 'react';
import { App } from './App';

// Create the mock store
import configureMockStore from 'redux-mock-store';
const mockStore = configureMockStore();

let wrapper, store;
beforeEach(() => {
  const initialState = {
    count: 1
  };
  store = mockStore(initialState);
  // Shallow render the container passing in the mock store
  wrapper = shallow(
      <App store={store} />
  );
});

describe('test the App component', () => {
  it('renders MainPage without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});