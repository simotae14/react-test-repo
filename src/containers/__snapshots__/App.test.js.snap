// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test the App component renders MainPage without crashing 1`] = `
<MainPage
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
/>
`;
